docs/components/modeler/forms/form-element-library/forms-element-library-textfield.md

```diff
@@ -15,11 +15,16 @@ A text field allowing the user to read and edit textual data.
 - **Key**: Binds the field to a form variable, see [data binding docs](../configuration/forms-config-data-binding.md).
 - **Default value**: Provides a default value for the text field in case no input data exists for the given key.
 - **Disabled**: Disables the text field, for use during development.
+- **Hide if**: [Expression](../../feel/language-guide/feel-expressions-introduction.md) to hide the text field.
 - **Validation**: Given that one of the following properties is set, the form will only submit when the respective condition is fulfilled. Otherwise, a validation error will be displayed.
   - **Required**: Text field must contain a value.
+  - **Regular expression validation**: Use predefined validation patterns (e.g. email)
```

Comment:
Maybe since it's the docs it's better to give an exhaustive list of the available?
```suggestion
  - **Regular expression validation**: Use predefined validation patterns. Available options are: `Email`, `Phone` and `Custom`
```

Reply:
Actually I realize one thing, should `regular expression validation` really be the name for this setting? Nothing about this setting requires us to implement it as a regex right? It's basically more of a 'quick validation' type of setting. I think the name may be confusing in the future, also we already have a very similarly named property in custom.

Maybe something we can look at this later, but @christian-konrad, @RomanKostka any input on this?

Reply:
I think we already have a rename in mind: https://github.com/bpmn-io/form-js/issues/454 (scheduled for spring cleaning).